### DISCUSSION TITLE
Is there any way of disabling word diff?

### DISCUSSION THREAD

**Discussion by caenrique** [neovim/neovim#29973] [Q&A]
I'm finding it annoying that most of my diffs doesn't show up as full line deleted/added when the only thing that remains the same is whitespace (the indentation)

![image](https://github.com/user-attachments/assets/9d051896-98de-4565-95c4-b635458b40e7)

the first line is what I would expect, since most of the line remains the same. Also the last line since it is a deletion (even if here the colors are also a bit misleading, but I know how to fix that), but for the rest, it's clearly only whitespace, and it makes it much less readable than a regular diff.

For reference, this is how it would look on github:

![image](https://github.com/user-attachments/assets/2ddaaa25-eab4-4466-9138-f9446b69ed93)

btw, this is clearly a naive example, I hope its clear that the issue gets much worse when working with bigger chunks of code and with more indentation

**Comment by Hamza12700** [comment]
You could use this plugin to show the git diff: [sindrets/diffview.nvim](https://github.com/sindrets/diffview.nvim)

**Reply by caenrique** [reply]
I am using it, but that plugin doesn't calculate the diff by itself, it's the same diff-mode under the hood, so it has the same issue. 
I'm showing here the issue running without any plugins or configuration to make it reproducible
